### PR TITLE
Dev marco

### PR DESCRIPTION
**Pull request by marcoplacenti**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,21 +1,18 @@
...
-transformers
-torch
-scikit-learn
-pytest
-yapf
-numpy
-matplotlib
-torchvision
-pytorch-lightning
-python-dotenv
```
Comment (stas97) [inline requirements.txt]: Let's not delete packages from the master `requirements.txt`, but only add to it. 
I see that you have many of the same, but we need to be careful and backwards compatible. So just add your packages to the end?

File: src/models/main.py [hunk 2]
```diff
@@ -47,17 +45,14 @@ def run(ctx, config_file):
 
         args = [config_file]
         config = ScriptRunConfig(source_directory='.',
-                                 script='src/models/train_model.py',
```
Comment (stas97) [inline src/models/main.py:50]: env like this should work then :)

**Review by stas97 — CHANGES_REQUESTED** [review]
Great work man!! :+1: 

Just add the packages to `requirements.txt` without deleting the previous ones and we good to go!

File: requirements.txt (hunk 1 above)
Comment (marcoplacenti) [reply requirements.txt]: Sorry, I messed it up while trying to fix the conflicts :)

**Pull request merged**